src/types/vendor.d.ts

```diff
@@ -0,0 +1,4 @@
+declare function t(app: string, string: string, vars?: { [key: string]: string }, count?: number, options?: EscapeOptions): string;
+
+declare module 'vue-material-design-icons/VideoOff'
```

Comment:
I think it makes sense to split this file into one for each dependency, and maybe one Nextcloud.d.ts for the core ones.